chpl/chpl-service/src/test/java/gov/healthit/chpl/validation/pendingListing/reviewer/PendingListingRequiredData2015ReviewerTest.java

```diff
@@ -322,6 +371,10 @@ private Boolean hasGenericTestTaskErrorMessage(final PendingCertifiedProductDTO
         return false;
     }
 
+    private String formatMessage(final String message, final String a, final String b, final String c, final String d) {
```

Comment:
Instead of another quite similar method here could you use java's variable arguments thing? It's like formatMessage(String... args) I have done something similar to what I'm thinking in the ErrorMessageUtil class. If it's not feasible that's fine but as I'm glancing over this it seems like it could work.